supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionBuilder.scala

```diff
@@ -63,7 +63,7 @@ object ActionBuilder {
     lazy val copyOnlyIfExists = s"if [[ -e $containerPath ]]; then $copy; fi"
 
     cloudSdkAction
-      .withCommand("/bin/sh", "-c", if (fileOutput.optional) copyOnlyIfExists else copy)
+      .withCommand("/bin/sh", "-c", if (fileOutput.optional || fileOutput.secondary) copyOnlyIfExists else copy)
```

Comment:
ToL: just wondering why we need a `secondary` boolean different from `optional` if eventually we just `||` them.
I was thinking of `optional` as meaning "this is an optional output" but not necessarily tied to being strictly of "optional type".
Again I'm fine with this, just curious if there was another reason I missed :)

Reply:
discussed in person